src/components/Filters/components/DateRangePicker/DateRangePicker.tsx

```diff
@@ -65,7 +66,7 @@ const DateRangePicker: React.FC<DateRangePickerProps> = ({
         endDate={endDate}
         maxDate={maxDate}
         minDate={startDate || minDate}
-        placeholderText="End date"
+        placeholderText={placeholder || 'End date'}
```

Comment:
Shouldn't we be able to pass separate placeholders for first and second field?

Reply:
Originally was planned to have placeholder only for `Number` and I added it to all components. I was aware of this use case for start and end date and I din't want to add exception if it is not required. But it is not big deal

Reply:
Updated